components/crud-web-apps/tensorboards/backend/app/utils.py

```diff
@@ -5,16 +5,34 @@ def parse_tensorboard(tensorboard):
     """
     Process the Tensorboard object and format it as the UI expects it.
     """
+    
+    if tensorboard.get("status", {}).get("readyReplicas", 0) == 1: 
+        phase = "ready"
+        message = "The Tensorboard server is ready to connect"
+    else:
+        phase = "unavailable"
+        message = "The Tensorboard server is currently unavailble"
 
     parsed_tensorboard = {
         "name": tensorboard["metadata"]["name"],
         "namespace": tensorboard["metadata"]["namespace"],
         "logspath": tensorboard["spec"]["logspath"],
         "age": helpers.get_age(tensorboard),
+        "status": {
```

Comment:
Could you use the helper function from the [status.py](https://github.com/kubeflow/kubeflow/blob/master/components/crud-web-apps/common/backend/kubeflow/kubeflow/crud_backend/status.py#L16) instead?